src/components/Button/Button.test.js

```diff
@@ -51,4 +51,14 @@ describe('<Button />', () => {
       expect(wrapper).toMatchSnapshot();
     });
   });
+
+  describe('disabled', () => {
+    beforeEach(() => {
+      wrapper = shallowWithTheme(<Button disabled>Hello world</Button>, theme);
+    });
+
+    it('renders correctly', () => {
+      expect(wrapper).toMatchSnapshot();
+    });
+  });
```

Comment:
I understand this is the pattern, but I think there's an opportunity to do a bit of clean up.
This pattern wastes a lot of space and can be rewritten as:

```suggestion
  test('disabled renders correctly', () => {
    wrapper = shallowWithTheme(<Button disabled>Hello world</Button>, theme);
    expect(wrapper).toMatchSnapshot();
  });
```

I also recommend using `mountWithTheme` so the styles are captured in the snapshot, as per my other comment.

Reply:
updated, also changed wrapper to a constant which I think is easier to read.